openlibrary/templates/type/edition/view.html

```diff
@@ -235,17 +235,16 @@ <h4 class="year">&mdash;
         </div>
 
     <div class="subjects">
-        <div class="subjects-content restricted-view">
-        $:macros.SubjectTags(work,"overview", ["Subjects"])
-        </div>
-        $:macros.ReadMore("subjects")
+      <div class="subjects-content">
+	$ component_times['SubjectsTags'] = time()
+        $:macros.SubjectTags(work, ["Subjects", "People", "Places", "Times"])
```

Comment:
```suggestion
        $ component_times['SubjectsTags'] = time()
        $:macros.SubjectTags(work, ["Subjects", "People", "Places", "Times"])
```
Replace tab with spaces.